test/standalone/Bucket/albedo_types.jl

```diff
@@ -91,298 +91,286 @@ FT = Float32
     @test p.bucket.α_sfc == next_alb_manual
 end
 
-# Add tests which use TempestRemap here -
```

Comment:
It's hard to read the diff, but the main change here is removing the `if !Sys.iswindows` check, right? I think everything else is independent of the remapper type

Reply:
Yes, correct!